nixos/modules/config/fonts/fontdir.nix

```
@@ -21,27 +25,42 @@ in
 {
 
   options = {
+    fonts.fontDir = {
 
-    fonts = {
-
-      enableFontDir = mkOption {
+      enable = mkOption {
         type = types.bool;
         default = false;
         description = ''
           Whether to create a directory with links to all fonts in
-          <filename>/run/current-system/sw/share/X11-fonts</filename>.
+          <filename>/run/current-system/sw/share/X11/fonts</filename>.
         '';
       };
 
-    };
+      decompressFonts = mkOption {
+        type = types.bool;
+        default = false;
```

```
    fonts.fontDir.decompressFonts = mkIf config.programs.xwayland.enable true;
```

This definition will easily cause a conflict. You could use `mkDefault` but It's better to simply do
```suggestion
        default = config.programs.xwayland.enable;
```

---

done